streamingvisitors/src/vespa/vsm/vsm/tokens_dfw.h

```diff
@@ -0,0 +1,28 @@
+// Copyright Vespa.ai. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#pragma once
+
+#include <vespa/searchsummary/docsummary/docsum_field_writer.h>
+#include <vespa/searchlib/query/query_normalization.h>
+
+namespace vsm {
+
+/*
+ * Class for writing annotated string field values from document as
```

Comment:
Please update the class comment to reflect how strings are converted to tokens (in vsm).